java-backend/src/main/java/org/kframework/backend/java/builtins/MetaK.java

```diff
@@ -201,6 +202,8 @@ public static KItem getKLabel(KItem kItem, TermContext context) {
     public static StringToken getKLabelString(Term term, TermContext context) {
         if (term instanceof KItem) {
             return StringToken.of(((KItem) term).kLabel().toString());
+        } else if (term instanceof KVariable){
+            return null;
```

Comment:
Could you explain in which case we need this?  An example?

Reply:
https://github.com/runtimeverification/verified-smart-contracts/blob/b9aa9543670a4b4bcf3d1450d44f66d028404cec/k-test/k-test-spec.ini#L59-L66